src/devices/Mbi5027/README.md

```diff
@@ -17,6 +17,11 @@ Note: The [MBI5168](http://archive.fairchip.com/pdf/MACROBLOCK/MBI5168.pdf) is a
 
 The [MBI5027 sample](samples/README.md) demonstrates how to use the shift register. The [generic shift register sample](../ShiftRegister/samples/README.md) is more extensive and is compatible with the MBI5027.
 
+## Documentation
+
+- Datasheet: http://archive.fairchip.com/pdf/MACROBLOCK/MBI5027.pdf
```

Comment:
I won't mark them all but would be perfect if you can adjust those raw URL with the correct pattern.

Markdown tip: In case you really want to show an URL, it should then be contained into < >